pkg/render/fluentd.go

```diff
@@ -675,6 +736,16 @@ func (c *fluentdComponent) envvars() []corev1.EnvVar {
 					}
 				}
 			}
+			// Enable TLS option for syslog forwarding.
+			if syslog.Encryption == operatorv1.EncryptionTLS && len(c.cfg.SysLogCredential.Certificate) != 0 {
+				envs = append(envs,
+					corev1.EnvVar{Name: "SYSLOG_TLS", Value: "true"},
+				)
+				envs = append(envs,
+					corev1.EnvVar{Name: "SYSLOG_VERIFY_MODE", Value: "OPENSSL::SSL::VERIFY_NONE"},
```

Comment:
This should be removed

Reply:
Done

Reply:
Updated the  "SYSLOG_VERIFY_MODE" to Value: "1" ("OPENSSL::SSL::VERIFY_PEER"). This option let the client (fluentd) verify the Cert provided by rsyslog server.